pkg/scheduler/partition_test.go

```diff
@@ -355,15 +355,15 @@ func TestRemoveNodeWithAllocations(t *testing.T) {
 	// wait for events to be processed
 	err = common.WaitFor(10*time.Millisecond, time.Second, func() bool {
 		fmt.Printf("checking event length: %d\n", eventSystem.Store.CountStoredEvents())
-		return eventSystem.Store.CountStoredEvents() == 1
+		return eventSystem.Store.CountStoredEvents() == 4 // 3 node alloc + 1 app event
```

Comment:
Created a follow-up to address this: https://issues.apache.org/jira/browse/YUNIKORN-1833.
The tests are becoming unmaintainable, this cannot be tested here.